source/rc.inetd/git-daemon/run

```diff
@@ -0,0 +1,15 @@
+#!/bin/execlineb -P
+
+########################
+# Start the git daemon #
+########################
+
+fdmove -c 2 1
+s6-setuidgid git
+emptyenv -p
+s6-envdir env
+multisubstitute {
+  importas -u CMD CMD
+  importas -usCd" " CMD_OPTS CMD_OPTS
+}
+emptyenv -c $CMD $CMD_OPTS
```

Comment:
Just a minor request, please change `emptyenv -c $CMD $CMD_OPTS` to
```
emptyenv -c
exec -a git-daemon $CMD $CMD_OPTS
```
To make it cleaner to look at when looking at the process name in `htop` or `ps` for example :D